src/components/Daos/DaosPage.tsx

```diff
@@ -66,13 +68,24 @@ class DaosPage extends React.Component<IProps, IState> {
     window.removeEventListener("resize", this.updateWindowDimensions);
   }
 
+
   private updateWindowDimensions = (_e: any) => {
     const nowMobile = window.innerWidth <= 550;
     if (nowMobile !== this.state.isMobile) {
       this.setState({ isMobile: nowMobile });
     }
   }
 
+  private registerForMonitoring = () => {
+    showSimpleMessage(
+      {
+        title: "BUIDLHub Proposal Monitoring",
+        body: <BHubReg />,
```

Comment:
Suddenly realized what you're doing here.  `showSimpleMessage` wasn't originally envisioned as being a container for any arbitrary form, rather to be just for simple OK/Cancel messages.  Normally for a complicated form with its own logic and idiosyncratic buttons, etc, we would go straight to `ModalPopup`, like `SimpleMessagePopup` itself is doing.

So I am nervous about asking `showSimpleMessage` to support this use case.  But I suppose if it is working then OK.

Reply:
Right. I was advised to use what's there vs. creating a whole new custom modal popup. I saw the "hideFooter" property but didn't think to just do what you suggested to make it actually work. But committed changes as you suggested.